internal/server/server.go

```diff
@@ -92,3 +98,61 @@ func (s *Server) Run() error {
 
 	return <-errs
 }
+
+type claims struct {
+	Roles []string `json:"roles"`
+	jwt.StandardClaims
+}
+
+const (
+	// adminRole defines the role for an administrator user.
+	adminRole = "admin"
```

Comment:
👀 You made a enum for the role values rather than just... a struct?

Reply:
Yeah